x/bank/module.go

```diff
@@ -157,7 +157,7 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 }
 
 // ConsensusVersion implements AppModule/ConsensusVersion.
-func (AppModule) ConsensusVersion() uint64 { return 2 }
+func (AppModule) ConsensusVersion() uint64 { return 3 }
```

Comment:
oh yeah, one thing I forgot to mention is to change update `RegisterServices` in this file: you need to also add a `err := cfg.RegisterMigration(types.ModuleName, 2, m.Migrate2to3)` (feel free to panic on error there)